draft-packages/tooltip/KaizenDraft/Tooltip/utils/isSemanticElement.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactNode } from "react"
+
+// Noting that this is contingent on our displayName for our components -  this something we control
+const allowedDisplayNames = ["Button", "IconButton"]
```

Comment:
@dougmacknz keens to get your thoughts on which components we should allow the Tooltip to describe (aria-describedby). I'm wondering if we should have table header row in here?

Reply:
So I checked the other places in Kaizen that we're using Tooltips:
- ✅ RTE's toggle buttons are fine
- ✅ Table headers are fine. There's an issue there where the header is only a button if the column is sortable, but out of scope for this work.

There's a couple of places in the filter bar and FilterMultiSelect world where it's not working, but it's actually a good thing because the tooltips are just repeating the button's label.

<img width="562" alt="image" src="https://github.com/cultureamp/kaizen-design-system/assets/1811583/4718de7e-204c-4082-8629-176febc146a0">
<img width="388" alt="image" src="https://github.com/cultureamp/kaizen-design-system/assets/1811583/4f363074-bbe5-47b3-b40e-7a7428227a0c">

Except for this Button Group one, which I think is using the same `FilterButtonBase` as the one above.
<img width="777" alt="image" src="https://github.com/cultureamp/kaizen-design-system/assets/1811583/37449fba-9345-40e2-9a48-632f4856f2fc">

So a bit of a difficult decision to make there, same as the `IconButton` problem.

Reply:
Digging into the code for the remove button, it looks like most of its accessible title is only derived image title prop on the icon (at least in our example). That said the consumers can pass in aria attributes into the `removeButtonProps`, which means the accessible label description should abel to be different, ie:

```
          <FilterButtonRemovable
            triggerButtonProps={{
              label: "Desserts",
            }}
            removeButtonProps={{
              onClick: () => undefined,
              "aria-label": "Remove desserts",
              tooltipText: "Could be different from the label ideally"
            }}
          />
```

Will produce this (with the FilterButtonBase added to the allow list):

![Screenshot 2023-07-19 at 3 07 05 pm](https://github.com/cultureamp/kaizen-design-system/assets/36558508/963434f9-9da7-456c-9bb1-ad788dae3405)


I'd say there's a larger question on how we handle a default accessible label for the button but I'd say we could save that for a separate PR since it relates specifically to the implementation of the `FilterButtonRemovable`